modules/remove-admin-user.adoc

```diff
@@ -3,19 +3,19 @@
 // * operating/manage-user-access/remove-admin-user.adoc
 :_mod-docs-content-type: PROCEDURE
 [id="remove_admin-user_{context}"]
-= Remove the admin user
+= Removing the admin user after installation
 
 After an authentication provider has been successfully created, it is strongly recommended to remove the `admin` user.
 
 Removing the `admin` user is dependent on the installation method of the {product-title-short} portal.
 Perform one of the following procedures:
```

Comment:
Out of scope to your work and thus optional, but I think this procedure module would benefit from a title somewhere:
```suggestion

.Procedure

Perform one of the following procedures:
```